File: dotnet/cafdemalihapa/Applications/Applications.cs
namespace cafdemalihapa.Applications
{
    public static class Applications
    {
        public static string Get()
        {
            var application = CommandLineArgs.GetByKey("--application");
            return application;
        }

        public static bool IsCafdemalihapa(this string application)
        {
            return application == "cafdemalihapa";
        }

        public static bool IsNotepadPlusPlusFileManagementApplication(this string application)
        {
            return application == "notepad-plus-plus-file-management";
        }

        public static bool IsIdeManagementApplication(this string application)
        {
            return application == "ide-management";
        }

        public static bool IsDirectoryManagementApplication(this string application)
        {
            return application == "directory-management";
        }
    }
}

File: dotnet/cafdemvers/zzq-cafdemwimu/cafdemwimu.console/Applications/Applications.fs
namespace cafdemwimu.console.Applications

open System.Runtime.CompilerServices
open cafdemwimu.console

[<Extension>]
type Applications =
    static member Get() =
        let application = CommandLineArgs.GetByKey("--application")
        application

    [<Extension>]
    static member IsCafdemalihapa(application: string) = application = "cafdemalihapa"

    [<Extension>]
    static member IsNotepadPlusPlusFileManagementApplication(application: string) = application = "notepad-plus-plus-file-management"

    [<Extension>]
    static member IsIdeManagementApplication(application: string) = application = "ide-management"

    [<Extension>]
    static member IsDirectoryManagementApplication(application: string) = application = "directory-management"
